sdk/data/aztables/cloud_config.go

```diff
@@ -12,12 +12,12 @@ const ServiceName cloud.ServiceName = "data/aztables"
 
 func init() {
 	cloud.AzureChina.Services[ServiceName] = cloud.ServiceConfiguration{
-		Audience: "https://storage.azure.cn",
+		Audience: "https://cosmos.azure.cn",
```

Comment:
@jhendrixMSFT, @chlowell - @xumou-ms is trying to fix a bug with token audiences. 

Azure Storage uses storage.azure.com in all clouds (apparently). Cosmos does use cloud specific audiences.

My question to you is what our contract with CloudConfig is - is it considered a contract breakage if the scope changes from being the storage scope, to being a cosmos scope instead?

Reply:
This just swaps out the audience for the token. The operations for the services are the same, apart from that, so I don't think there's anything for Cosmos to do here. This is purely a client-side thing.